src/app/profile/view.nim

```diff
@@ -120,6 +121,14 @@ QtObject:
   QtProperty[QVariant] ens:
     read = getEnsManager
 
+  proc changePassword(self: ProfileView, password: string, newPassword: string): bool {.slot.} =
+    try:
+      if not status_accounts.changeDatabasePassword(self.profile.address, password, newPassword):
```

Comment:
We should not call libstatus functions directly. `changeDatabasePassword` should be added to the accounts model in src/status/accounts.nim, and then the new proc in that model should call libstatus.